doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1230,6 +1239,7 @@ Documentation
   * The :ref:`dfu_tools_mcumgr_cli` page after it was removed from the Zephyr repository.
   * The :ref:`ug_nrf54h20_suit_soc_binaries` page.
   * The :ref:`ug_nrf54h20_suit_push` page documentating the SUIT push model-based update process.
+  * The :ref:`app_secure_approtect` section to the :ref:`app_approtect` page.
```

Comment:
There is more than just one new section.

Reply:
The other sections are subsection of this.